goc/share/oasis-batch-worker

```diff
@@ -39,8 +39,10 @@ if [ $vo ]; then
        echo "Update initiated: $now">/tmp/update.details
        now=`date +%s`
        echo "Update unix time: $now">>/tmp/update.details
-       if [ ! -f $cert ] ; then
-	   echo "No proxy found for user $user" >>/tmp/update.details
+       if [ -n "$CILOGONID ]; then
```

Comment:
```suggestion
       if [ -n "$CILOGONID" ]; then
```

or equivalently, just

```suggestion
       if [[ $CILOGONID ]]; then
```

Reply:
Oh dear, I guess I didn't test that.  I wrote it awhile ago and forgot I hadn't tested it.  Thanks for catching it.